service/project.go

```diff
@@ -366,6 +366,26 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 			uis.LoggedError(w, r, http.StatusBadRequest, errors.Errorf("Cannot enable Commit Queue without github webhooks enabled for the project"))
 			return
 		}
+		// if no new commit queue aliases, verify there are pre-existing commit queue aliases
+		if len(responseRef.CommitQueueAliases) == 0 {
+			aliases, err := model.FindAliasInProject(responseRef.Identifier, evergreen.CommitQueueAlias)
+			if err != nil {
+				uis.LoggedError(w, r, http.StatusInternalServerError, errors.Wrapf(err, "error checking for commit queue aliases"))
+				return
+			}
+
+			remainingAliases := 0
+			for _, a := range aliases {
+				// only consider aliases that won't be deleted
+				if !util.StringSliceContains(responseRef.DeleteAliases, a.ID.String()) {
```

Comment:
The id in Angular's scope looks like it's just the hex digits.

![Screen Shot 2019-11-26 at 11 49 46 AM](https://user-images.githubusercontent.com/17027265/69654552-fefb7600-1042-11ea-81a1-aa182757153c.png)